api/proto/message/message.proto

```diff
@@ -82,11 +82,12 @@ message StakingRequest {
 message ConsensusRequest {
   uint64 view_id = 1;
   uint64 block_num = 2;
-  uint32 shard_id = 3;
-  bytes block_hash = 4;
-  bytes block = 5;
-  bytes sender_pubkey = 6;
-  bytes payload = 7;
+  uint64 epoch = 3;
```

Comment:
cannot change protobuf def -

Reply:
hmm, why not?

Reply:
actually, guess it would make rolling upgrades not possible.

Reply:
yup, can only add field new field number, can't change existing numbers.

Reply:
cool latest is fine then?